src/05-invoicing/invoices.apib

```diff
@@ -144,7 +148,9 @@ Get details for a single invoice.
                                     + Members
                                         + percentage
                                 + value: 15.00 (number)
+                                + absolute_value (Money)
             + total (object)
+                + tax_and_discounts_inclusive (Money)
```

Comment:
PP : `discounted`

Reply:
PP : put the totals in the same order as what we would on the printed invoice:

- exclusive
- discounted
- taxes
- inclusive
- payable
- due

Not sure yet where to put the taxes